### ISSUE TITLE
Organizer Update Competition

### ISSUE THREAD

**Issue by yubing24** [DancesportSoftware/das-frontend#110]
Organizers should be able to make changes to competitions, including but not limited to:
- [x] Change the venue of competition
- [x] Change the status of competition(pre-registration, open-registration, close-registration, in-progress, processing, closed, cancelled)
- [x] Change the start/end date of competition

Competition Management Console ("the Console") is introduced in commit d364108a117207b0434e2948660875fc4cbb2ace. Functions listed above should only be available in the Console. Organizer main UI should mostly remain the same for administrative purposes, and competition-related management should be moved to the Console.

Functions listed above should be implemented under the Settings page in the Console.

API method: `PUT /api/v1.0/organizer/competition`

Minimum JSON paylod in request body: 
```
{
    "competition: 2,
    "street": string, optional,
    "status": number, optional
}
```

Valid status can be obtained from `/api/competition/status`

**Comment by shadowlion** [comment]
@yubing24 This be done.

**Comment by yubing24** [comment]
Implemented in #117.
